sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/SchemaRegistryAvroObjectSerializerLiveTests.cs

```diff
@@ -212,6 +214,24 @@ public async Task CanUseEncoderWithEventData()
             // the serialized Avro data will be stored in the EventBody
             Console.WriteLine(eventData.EventBody);
 #endif
+
+            // construct a publisher and publish the events to our event hub
+#if SNIPPET
+            var fullyQualifiedNamespace = "<< FULLY-QUALIFIED EVENT HUBS NAMESPACE (like something.servicebus.windows.net) >>";
+            var eventHubName = "<< NAME OF THE EVENT HUB >>";
+            var credential = new DefaultAzureCredential();
+#else
+            var fullyQualifiedNamespace = TestEnvironment.SchemaRegistryEndpoint;
+            var eventHubName = TestEnvironment.EventHubName;
+            var credential = TestEnvironment.Credential;
+#endif
+            await using var producer = new EventHubProducerClient(fullyQualifiedNamespace, eventHubName, credential);
+            using EventDataBatch eventBatch = await producer.CreateBatchAsync();
+            if (!eventBatch.TryAdd(eventData))
+            {
+                throw new InvalidOperationException("Payload is too large to be sent to Event Hubs service.");
+            }
+            await producer.SendAsync(eventBatch);
```

Comment:
```suggestion
            // because we're sending a single event, we'll avoid the overhead of creating a batch. 
            await producer.SendAsync(new EventData[] { eventData });
```

I'm thinking it would be simpler to show the enumerable overload.   Since we've got a single event, we'll know exactly what failed if it was too large, so the batch doesn't buy us anything.  We probably want to consider tweaking the comment a bit, though.